numpy/core/fromnumeric.py

```diff
@@ -1980,25 +1980,26 @@ def shape(a):
 
     See Also
     --------
-    len
+    len : ``len(a)`` is equivalent to ``np.shape(a)[0]`` for 1+-dim arrays.
```

Comment:
This may be kind of a nit but I'd say instead
```suggestion
    len : ``len(a)`` is equivalent to ``np.shape(a)[0]`` for N-D arrays with ``N>1``.
```